jbt-it/neomdb
# #325 add image support

- Added get-image route
- Added post-image route
- Added image retrieval to profile page
- Added image upload to profile page

**How to test:**
1. Start server & client
2. Navigate to http://localhost:3030/api/docs/ and test if the following two routes behave as expected:
- http://localhost:3030/api/docs/#/Members/GetMemberImage
- http://localhost:3030/api/docs/#/Members/SaveImage
3. Navigate to http://localhost:3000, log in and navigate to own profile page.
4. Try to upload an image by clicking on your profile image
5. Navigate to another member profile and try to upload an image (this should not be possible)
